backend/sirius-web-spring-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -27,6 +27,7 @@ type Diagram implements Representation {
   nodes: [Node!]!
   edges: [Edge!]!
   toolSections: [ToolSection!]!
+  unsynchronizedDiagramElementsDescriptionIds: [ID!]!
```

Comment:
As discussed this morning, instead of adding such field, it would be better to have the relevant information on `NodeDescription`. The current approach has several drawbacks:

- Since it is a computed information, it requires everyone using Sirius Components to provide a data fetcher with this behavior
- We don't want to start adding custom fields for small use cases like that